### ISSUE TITLE
Assessment Test

### ISSUE THREAD

**Issue by Passchendaele** [Pierian-Data/Complete-Python-3-Bootcamp#406]
I am not able to write in any of the spaces. Also, I am not seeing a download zip option for the test

**Comment by jefhar** [comment]
Did you read section 1 chapter 3 of the course?

**Comment by Passchendaele** [comment]
Yes...but here is what I am getting...no option to download the test
![Capture](https://user-images.githubusercontent.com/96503937/147159985-24a50d03-87e7-4f38-aa5c-406694bbe9a5.JPG)

**Comment by Passchendaele** [comment]
There seems to be a lack of desire to help here...I also noticed this in other past comments

**Comment by Pierian-Data** [comment]
Please post questions to the course Q&A forum as mentioned in the video
lectures. Also please make sure to watch the first few videos of the
course, based on your post here and questions, it's clear you skipped some
important set up lectures.

**Comment by poojaalugoju** [comment]
how do I acess this exam

**Comment by poojaalugoju** [comment]
how do I access this exam

**Comment by PrajwalSanak** [comment]
Download the whole course code...

**Comment by poojaalugoju** [comment]
i have downloaded but when i'am trying to open the file its showing some code 
![Screenshot (649)](https://user-images.githubusercontent.com/97930387/149922973-b82ce419-dbb7-413e-a350-fdf7fb1ff18e.png)

**Comment by PrajwalSanak** [comment]
Open in Jupiter notebook.

**Comment by PrajwalSanak** [comment]
Open in Jupiter notebook.

**Comment by Elisa0426** [comment]
It doesn't work...

**Comment by PrajwalSanak** [comment]
Send me the screenshot
